plots/ab-screenshot/analyze.js

```diff
@@ -0,0 +1,217 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const opn = require('opn');
+const args = require('yargs').argv;
+
+const Metrics = require('../../lighthouse-core/lib/traces/pwmetrics-events');
+
+const constants = require('../constants');
+const utils = require('../utils');
+
+/**
+ * Do a comparison analysis of two batches of runs. This helps you compare how a change in
+ * lighthouse can affect perf results in real-world sites.
+ *
+ * Example usage:
+ * node analyze.js -a ./out-first -b ./out-second
```

Comment:
I originally didn't think about this use case because then you're comparing two traces with the same configuration (e.g. lighthouse, chrome version) and in that case, I think the end goal could be to either A) visualize the variance between runs when all factors are held constant or B) compare multiple sites (e.g. a PM wants to compare their site vs. 3 competitors).

These seem like nice to have cases but not essential. Let's defer them until later?